src/views/application.tsx

```diff
@@ -13,6 +13,7 @@ import {LeftHeavyFlamechartView, ChronoFlamechartView} from './flamechart-view-c
 import {CanvasContext} from '../gl/canvas-context'
 import {Graphics} from '../gl/graphics'
 import {Toolbar} from './toolbar'
+import {importJavascriptSymbolMap} from '../lib/js-source-map'
```

Comment:
I try to use code-splitting fairly aggressively in speedscope to keep the JS needed to boot the application as small as possible. To get this to be a part of a different chunk (which can be lazy-loaded), you can use the pattern below with `const importModule = import('../import')`. So in this case:

```
const jsSourceMapModule = import('../lib/js-source-map')
jsSourceMapModule.then(() => {})
```

And then put that below the `importModule` bits below to load it with a lower priority than the `import.ts` module.

This gives you a promise which resolves as the module. So you should be able to something like the following inline inside of the `loadFromFile` function:

`const {importJavascriptSymbolMap} = await jsSourceMapModule`